src/dev.sh

```diff
@@ -32,6 +36,7 @@ function failed()
 {
    local error=${1:-Undefined error}
    echo "Failed: $error" >&2
+   cd $CURRENT_DIR
```

Comment:
I think it would be a little cleaner to pushd and popd rather than introduce new variables that aren't needed anywhere else.